content/blog/centralized-policy-violations/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Introducing the Centralized Policy Violations Page: Streamlined Compliance Management"
+allow_long_title: true
+date: 2024-08-26T09:00:00-07:00
+draft: false
+meta_desc: The new Centralized Policy Violations page in Pulumi offers enhanced visibility and control by consolidating all policy violations.
+meta_image: meta.png
+authors:
+    - meagan-cojocar
+tags:
+    - releases
+    - policy
+---
+
+We’re excited to introduce a new page in Pulumi Cloud designed to simplify policy management and compliance: the centralized Policy Violations page. This powerful addition provides a single, comprehensive view of all policy violations across your organization, helping you maintain control and visibility over your infrastructure.
+
+Pulumi’s Policy as Code (PaC) feature, [CrossGuard](/docs/using-pulumi/crossguard), allows you to define and enforce policies for your infrastructure using familiar programming languages. By integrating these policies directly into your infrastructure code, you can automate compliance checks, gate deployments to ensure security best practices, and maintain governance across all your providers.
+
+**Note:** Pulumi CrossGuard is available in the Pulumi Business Critical edition.
+
+## Centralized Policy Violations: Why It Matters
+
+In large and complex environments, ensuring compliance across multiple projects and stacks can be challenging. The new Policy Violations page addresses this challenge by consolidating all violations into one place, allowing you to quickly assess the overall compliance of your infrastructure and separate signal from noise. This streamlined view not only saves time but also reduces the risk of missing critical issues.
+
+By having all violations on a single page, organization admins can efficiently prioritize remediation efforts and identify patterns in non-compliance. They can triage the violations and send the critical ones to teams to resolve.
+
+{{< video title="Centralized Policy Violations Overview" src="policy-violations-overview.mp4" autoplay="false" loop="false" >}}
+
+## Grouping Violations: Enhanced Insights Across Multiple Dimensions
+
+One of the standout features of the new page is the ability to group violations by various fields, including Policy Pack, Policy, Project, Stack, Resource, Reason, Enforcement Level, Version, and Violation Date. This flexibility allows you to gain deeper insights into your infrastructure’s compliance landscape.
```

Comment:
Do we want to actually list all the fields you can group on, as this will likely evolve?  Should we just list a few with words like "to name a few" to indicate there are more?